### PR TITLE
expired listing on sell page

### PR DESCRIPTION
**Pull request by vikas-cis-zz**
https://particl.atlassian.net/browse/PBD-227

### CODE REVIEW

**Comment by vikas-cis-zz** [comment]
@AllienWorks  Can you please have a look regarding the expired listing , I think it require some css stuff on sell page below is the screenshot how it looks currently 


screen shot 2019-02-18 at 8 37 56 pm

Thanks

**Comment by AllienWorks** [comment]
..aaand done!

screenshot from 2019-02-14 12-27-56

**Comment by vikas-cis-zz** [comment]
Thank you @AllienWorks :)

File: src/app/core/market/api/template/template.model.ts [hunk 1]
```diff
@@ -179,22 +179,34 @@ export class Template {
...
+    return  this.checkListingItems ? 'Expires ' + new DateFormatter(
+      new Date(this.object.ListingItems[0].expiredAt)
+      ).dateFormatter(false).substr(0, 16) : '';
   }
 
   get isAboutToExpire(): Boolean {
 
     // 86400000 m seconds in one day.
-    return (this.object.expiredAt > +new Date() && ((this.object.expiredAt - +new Date()) <= 86400000));
+    return (this.object.expiredAt < +new Date() && ((this.object.expiredAt - +new Date()) <= 86400000));
```
Comment (zaSmilingIdiot) [inline src/app/core/market/api/template/template.model.ts]: Was this in error, or intentional? It changes the meaning of the condition to indicate that the item has already expired.

Comment (vikas-cis-zz) [reply src/app/core/market/api/template/template.model.ts]: @zaSmilingIdiot possibly I was doing some testing and pushed the previous change my bad :( 

Thanks

**Review by zaSmilingIdiot — APPROVED** [review]

**Pull request merged**